utils/constants.ts

```diff
@@ -26,32 +26,17 @@ export const NftCollections = [
     name: "Ducks everywhere",
     externalLink: "https://briq.construction/product/ducks_everywhere",
```

Comment:
change this link to https://unframed.co/collection/0x04fa864a706e3403fd17ac8df307f22eafa21b778b73353abf69a622e47a2003